README.md

```diff
@@ -659,6 +659,56 @@ registration time instead of `describe`. If this could cause problems, either
 implement a proper `describe`, or if that's not practical have `describe`
 return an empty list.
 
+### DropwizardExports Collector
+
+DropwizardExports collector is available to proxy metrics from Dropwizard.
+
+```java
+// Dropwizard MetricRegistry
+MetricRegistry metricRegistry = new MetricRegistry();
+new DropwizardExports(metricRegistry).register();
+```
+
+By default Dropwizard metrics are translated to Prometheus sample sanitizing their names, i.e. replacing unsupported chars with `_`, for example:
+```
+Dropwizard metric name:
+org.company.controller.save.status.400
+Prometheus metric:
+org_company_controller__save_status_400
+```
+
+It is also possible add custom labels and name to newly created `Sample`s by using a `CustomMappingSampleBuilder` with custom `MapperConfig`s:
+
+```java
+// Dropwizard MetricRegistry
+MetricRegistry metricRegistry = new MetricRegistry();
+MapperConfig config = new MapperConfig();
```

Comment:
This is a bit long, would it be better to take all this in the constructor?

Do you have plans around offering the yaml config format that graphite/statsd exporters do?

Reply:
I've set MapperConfig's attributes explicitly in the docs for clarity. MapperConfig has a multi-arg constructor.
Do you mean adding support for `help` field, `regex` match type and `$n` placeholders?